json struct.py

```diff
@@ -67,7 +67,7 @@ def recursive_generate_header_content(configuration, tabs):
             plug_text += inner_text
         else:
             plug_text += tabs + get_header_line(key, value)
-    return plug_text[:-1] # Remove last newline
```

Comment:
This is the plug text. It's plugged somewhere in the middle. 
If you'll look in generate_header just below, it's plugged instead of PLACE_HOLDER.

Reply:
I'm aware. I forget the details, but I do remember that this change seemed necessary for the script to work correctly with the input file I used. I think the removal of this line might have something to do with the addition of `old_chain = chain`, a few lines further down. That's my best guess, anyway.